packages/babel-plugin-transform-es2015-block-scoping/test/fixtures/general/switch/expected.js

```diff
@@ -9,3 +11,17 @@ switch (a) {
     var d = 3;
     break;
 }
+
+switch (true) {
+  case true:
+    var _a = false;
+    var _b = true;
+    var _c = 4;
+    var _d = 5;
+
+  case false:
+    class _e {}
```

Comment:
I'm just being nit-picky :P. If the objective of this transform is to process block scoping, it surprised me to see a structure left that is inherently block scoped, like a class declaration. I'm sure it was left on purpose since there's no actual reason to bother converting it here, since there isn't going to be an environment that has implemented classes but not implemented block scoping. Just commenting that it's interesting, this is what I get for doing code review past midnight.